src/__tests__/vue-hcaptcha.test.js

```diff
@@ -0,0 +1,141 @@
+import VueHcaptcha from '@/main';
+import {DEBUG_SITE_KEY, getTestWrapper} from './utils';
+import {getMockedHcaptcha, MOCK_EKEY, MOCK_TOKEN, MOCK_WIDGET_ID} from './hcaptcha.mock';
+import {loadApiEndpointIfNotAlready} from '@/hcaptcha-script';
+import {nextTick} from 'vue';
+
+/*global describe, test, expect, beforeEach, jest*/
+
+jest.mock('@/hcaptcha-script', () => ({
+    loadApiEndpointIfNotAlready: jest.fn(() => Promise.resolve())
+}));
+
+describe('VueHCaptcha', () => {
```

Comment:
Any way to validate that the rendering (invisible, compact, dark mode) is correct?

Reply:
There is a test `el renders after api loads and a widget id is set` which asserts that the passed properties are used in rendering. 

`src/__tests__/vue-hcaptcha.test.js` lines 113-118